include/invert_quda.h

```diff
@@ -812,6 +837,37 @@ namespace quda {
     void operator()(ColorSpinorField &out, ColorSpinorField &in);
   };
 
+  class CACGNE : public CACG {
+
+  private:
+    DiracMMdag mmdag;
+    DiracMMdag mmdagSloppy;
+    ColorSpinorField *xp;
+    ColorSpinorField *yp;
```

Comment:
clang-format suggested the following change:


```
-    ColorSpinorField *xp;
-    ColorSpinorField *yp;
+    ColorSpinorField* xp;
+    ColorSpinorField* yp;
```

<!-- identifier: clang-format -->
<!-- File: include/invert_quda.h -->